client/my-sites/checklist/wpcom-checklist/component.jsx

```diff
@@ -897,10 +897,13 @@ class WpcomChecklistComponent extends PureComponent {
 				title={ translate( 'Update your homepage' ) }
 				completedTitle={ translate( 'You updated your homepage' ) }
 				bannerImageSrc="/calypso/images/stats/tasks/personalize-your-site.svg"
-				completedButtonText={ translate( 'Change' ) }
+				completedButtonText={ translate( 'Edit homepage' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **18** times:
`translate( 'Edit Homepage' )` *ES Score: 10*